### PR TITLE
Issue #226: Add Malware information to overview

### PR DESCRIPTION
**Pull request by DonNateR**
Fixes issue #226 

I was going to add this to the integration docs but I think it makes more sense either in the "Detections" page, or the "Overview" page since it contains a high-level overview of what malware is. 

However, I did put the integration and configuration steps on the install page.

Let me know if that makes sense to you @jmikell821

### CODE REVIEW

File: docs/es-overview.asciidoc [hunk 1]
```diff
@@ -121,3 +121,31 @@ to better analyze, visualize, and correlate the data represented in their
...
+To determine if a file is malicious or benign, a machine learning model looks for static attributes of files (without executing
+the file) that include file structure, layout, and content. This includes information such as file header data, imports, exports,
+section names, and file size. These attributes are extracted from millions of benign and malicious file samples, which then
+are passed to a machine-learning algorithm that distinguishes a benign file from a malicious one. The machine learning
+model is updated as new data is procured and analyzed.
+
+[discrete]
+=== Threshold
+
+A malware threshold determines the action the agent should take if malware is detected. The Elastic Agent uses a recommended threshold level that generates a balanced number of alerts with a low probability of undetected malware. This threshold also minimizes the number of false positive alerts. For information on how to enable malware protection with the Elastic Agent, see <<install-endpoint, Configure and install Elastic Endpoint Integration (beta)>>.
```
Comment (benskelker) [inline docs/es-overview.asciidoc:151]: Just wondering, if we mention a malware threshold, can users change it?

File: docs/es-overview.asciidoc [hunk 2]
```diff
@@ -32,13 +32,13 @@ Endpoint Security agent integration:
 ** <<install-endpoint, {es-sec-endpoint}>> - Elastic agent integration that
 protects your hosts against malware and ships these data sets:
```
Comment (benskelker) [inline docs/es-overview.asciidoc:33]: Maybe add a link here from `against malware` to your new section?

**Review by benskelker — APPROVED** [review]
LGTM - thanks

File: docs/es-overview.asciidoc (hunk 1 above)
Comment (DonNateR) [reply docs/es-overview.asciidoc:151]: You can change whether to "detect" or "protect" but I don't think you can change the threshold. Adding detect vs prevent instructions in my next commit.

**Comment by jmikell821** [comment]
@DonNateR I think it's a little TMI to add this to the Overview section, though I see what you did that. What do you think about making this an `H2` under `Detections and Alerts`?

**Comment by DonNateR** [comment]
@jmikell821: Sounds good to me. I'll update the PR now.

**Comment by benskelker** [comment]
@DonNateR @jmikell821 - I think this should be a section under here: [Additional Elastic Endpoint Security information](https://www.elastic.co/guide/en/security/current/es-overview.html#_additional_elastic_endpoint_security_information). This is more personal opinion than anything else - so I'm fine with whatever you decide on.

**Comment by jmikell821** [comment]
@benskelker thanks Ben. I think it can go either way but I didn't want this section to get too long. I'll ask Braden once we get everything merged.

**Pull request merged**